pkg/apis/apps/types.go

```diff
@@ -97,6 +97,26 @@ type RollingUpdateStatefulSetStrategy struct {
 	Partition int32
 }
 
+// PersistentVolumeClaimDeletePolicyType is a string enumeration of the policies for
+// deletion of VolumeClaimTemplates when the controllering StatefulSet is deleted or
```

Comment:
nit: controllering => controlling

Reply:
done

Reply:
ack, something went wrong with the commit splitting, trying again.

Reply:
okay, should be good to go.